general/jobSubmission/initiateDataFolders.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
This file is in "jobSubmission" which might imply that the script it to be submitted via `sbatch`. Do you think this is a problem? This script clearly doesn't need to be submitted via `sbatch`, its basically a one liner.

Reply:
ye I wasn't sure where to put it